hail/src/test/scala/is/hail/expr/ir/LocusFunctionsSuite.scala

```diff
@@ -81,4 +82,31 @@ class LocusFunctionsSuite extends HailSuite {
 
     assertEvalsTo(ir, Row(Locus("1", 1, ReferenceGenome.GRCh37), Locus("chr1", 1, ReferenceGenome.GRCh38)))
   }
+
+  @Test def testMakeInterval() {
```

Comment:
this test doesn't show the behavior that led to the bug, but does demonstrates the behavior of the output value, in  line with all of our other function registry tests.